packages/adapter-vercel/index.js

```diff
@@ -31,7 +31,8 @@ export default function () {
 				entryPoints: ['.svelte-kit/vercel/entry.js'],
 				outfile: join(dirs.lambda, 'index.js'),
 				bundle: true,
-				platform: 'node'
+				platform: 'node',
+				...config?.kit?.vite()?.esbuild
```

Comment:
I think that reusing `vite`'s `esbuild` config here is not the best way. People way want to configure the two separately. E.g. see https://github.com/sveltejs/kit/pull/1692#issuecomment-865615019